ateliermecanique-ws/src/main/java/com/champlain/ateliermecaniquews/customerinvoicemanagementsubdomain/datalayer/CustomerInvoice.java

```diff
@@ -1,4 +1,48 @@
 package com.champlain.ateliermecaniquews.customerinvoicemanagementsubdomain.datalayer;
 
+import com.fasterxml.jackson.annotation.JsonFormat;
+import jakarta.persistence.*;
+import lombok.Data;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+@Entity
+@Table(name = "invoices")
+@Data
 public class CustomerInvoice {
+

```

Comment:
We should also have a list of services.

Reply:
you're right, I had just linked the appointment id in the table so you could view services in that appointment, but you're right especially since we want to allow the admin to modify the services. we'll talk about it in scrum meeting.